src/router.js

```diff
@@ -0,0 +1,62 @@
+import express from 'express';
+import { checkSchema } from 'express-validator';
+import db from './db';
+import postSchema from './validation/schema_postMotivator';
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  db.models.motivator.findAll({
+    include: [
+      {
+        model: db.models.motivatorContent,
+      },
+      {
+        model: db.models.motivatorResult,
+        where: {
+          user_id: req.user.uid,
+        },
+        limit: 1,
+        include: [
+          {
+            model: db.models.motivatorResultInput,
+          },
+        ],
+      },
+    ],
+  }).then((query) => {
+    const response = query.map((item) => item.toJSON())
+      .map((item) => {
+        const content = item.MotivatorContents.map((x) => JSON.parse(x.content));
+
+        const result = item.MotivatorResults.map(({
+          motivator_id, user_id, MotivatorResultInputs, ...resultItem
+        }) => ({ ...resultItem }));
+
+        const inputs = item.MotivatorResults[0].MotivatorResultInputs
+          .map(({ value }) => ({ ...JSON.parse(value) }));
+
+        return {
+          ...item, content, result: result[0], inputs,
+        };
+      })
+      .map(({ MotivatorContents, MotivatorResults, ...itemWithoutContents }) => ({
+        ...itemWithoutContents,
+      }));
```

Comment:
You could e.g. use an `Object.keys()` array and a `.filter()` to achieve the same functionality with more clarity. But regarding your comment in the PR opening, thats not a stopper for merging.